packages/ruby/index.ts

```diff
@@ -142,14 +139,7 @@ export async function build({
       } else {
         // try installing. this won't work if native extesions are required.
         // if that's the case, gems should be vendored locally before deploying.
-        try {
-          await bundleInstall(bundlerPath, bundleDir, gemfilePath);
-        } catch (err) {
-          debug(
-            'unable to build gems from Gemfile. vendor the gems locally with "bundle install --deployment" and retry.'
-          );
-          throw err;
```

Comment:
This was a double try/catch, so I removed this one in favor of the inner one in `bundleInstall`